languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/introduction.md

```diff
@@ -16,7 +11,8 @@ ignored by the computer. Single-line comments begin with one or more semi-colons
 Where the comment is being used to indicate what value is returned by Common
 Lisp after running the code on that line.
 
-## S-Expressions
+## expressions
+## cons
```

Comment:
It looks like one of these concepts is missing a description.

Reply:
Uh yeah. I left them as sort of co-headers instead of attempting to rewrite the section to untangle them. Or change the concept tree to perhaps remove 'expressions'...

Reply:
Right. I would suggest adding a TODO somewhere, to make it clear that something needs to be done.

```suggestion
## expressions

TODO: Untangle section below or possible remove "expressions" concept

## cons
```